src/Controller/Admin/CustomerController.php

```diff
@@ -28,6 +28,23 @@
  */
 class CustomerController extends AbstractController
 {
+    /**
+     * Default form settings from kinai.yml/local.yml
```

Comment:
Can you please remove complete line or at least the part `from kinai.yml/local.yml`?